### DISCUSSION TITLE
Ability for user to delete content from drive?

### DISCUSSION THREAD

**Discussion by evlo** [sct/overseerr#2335] [Help]
Hi, 

I'm currently using ombi, i'm running super small server, just me and 1 user, don't plan on making it any bigger, but I would like for admin/user to be able to delete files from drive after they watched what they requested. Can this be done in overseerr?

**Comment by TheCatLady** [comment]
Overseerr will not allow deletion of media files within its interface.

**Reply by evlo** [reply]
is there something like best practices/case how people achieve this behaviour? Or most just hoards the data so it is low prio?

**Reply by TheCatLady** [reply]
This just isn't something that will be handled by Overseerr, so if you want to delete items, it needs to be done outside of Overseerr.

That said, if items are deleted, they currently need to be manually cleared/removed from Overseerr because of #377.  Syncing media deletes is planned, but there is no ETA on the feature being implemented.

I personally do not delete media.
